modules/aruco/tutorials/aruco_detection/aruco_detection.markdown

```diff
@@ -41,48 +41,49 @@ It must be noted that a marker can be found rotated in the environment, however,
 process needs to be able to determine its original rotation, so that each corner is identified
 unequivocally. This is also done based on the binary codification.
 
-A dictionary of markers is a set of markers that are considered in an specific application. It is
+A dictionary of markers is the set of markers that are considered in a specific application. It is
 simply the list of binary codifications of each of its markers.
 
 The main properties of a dictionary are the dictionary size and the marker size.
 
-- The dictionary size is the number of markers that composed the dictionary.
+- The dictionary size is the number of markers that compose the dictionary.
 - The marker size is the size of those markers (the number of bits).
 
 The aruco module includes some predefined dictionaries covering a range of different dictionary
 sizes and marker sizes.
 
 One may think that the marker id is the number obtained from converting the binary codification to
 a decimal base number. However, this is not possible since for high marker sizes the number of bits
-is too high and managing so huge numbers is not practical. Instead, a marker id is simply
-the marker index inside the dictionary it belongs to. For instance, the first 5 markers inside a
-dictionary has the ids: 0, 1, 2, 3 and 4.
+is too high and managing such huge numbers is not practical. Instead, a marker id is simply
+the marker index within the dictionary it belongs to. For instance, the first 5 markers in a
+dictionary have the ids: 0, 1, 2, 3 and 4.
 
-More information about dictionaries is provided in the "Selecting a dictionary" section.
+More information about dictionaries is provided in the “Selecting a dictionary” section.
 
 
 Marker Creation
 ------
 
 Before their detection, markers need to be printed in order to be placed in the environment.
-Marker images can be generated using the ```drawMarker()``` function.
+Marker images can be generated using the `drawMarker()` function.
 
 For example, lets analyze the following call:
 
 @code{.cpp}
     cv::Mat markerImage;
     cv::Ptr<cv::aruco::Dictionary> dictionary = cv::aruco::getPredefinedDictionary(cv::aruco::DICT_6X6_250);
     cv::aruco::drawMarker(dictionary, 23, 200, markerImage, 1);
+    cv::imwrite("marker23.jpg", markerImage);
```

Comment:
Please use PNG format for markers (it is black-white image)

Reply:
Done. And replaced marker23.jpg with marker23.png in the images folder.